db/migrate/20160531170531_remove_duplicate_aspect_visibilities.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveDuplicateAspectVisibilities < ActiveRecord::Migration
+  def up
+    min_ids = AspectVisibility.group(:aspect_id, :shareable_id, :shareable_type).pluck(:id)
+    AspectVisibility.where.not(id: min_ids).destroy_all
```

Comment:
my production-db contains 672921 entries (with 5 duplicates), so you want so use a query with `WHERE (aspect_visibilities.id NOT IN (672916 IDs))`?

This raises a `ActiveRecord::StatementInvalid: Mysql2::Error: MySQL server has gone away` on my pod :scream_cat: I apologized and promised that I never ever do this again, when he comes back, so everything is fine :smiley_cat: but we need another query for this, where my mysql server doesn't walk away. :wink: